### PR TITLE
Added host_name and host_ip to MQTT messages

### PR DESCRIPTION
**Pull request by MaxVRAM**
As title of commit. Import socket library to include host ip and host name to MQTT packet.

Requested in feature request issue #63

### CODE REVIEW

**Comment by MaxVRAM** [comment]
Tested and running on RasPi 4 running Ubuntu Server 64-bit

File: src/system_sensors.py [hunk 1]
```diff
@@ -177,6 +179,11 @@ def get_wifi_strength():  # check_output(["/proc/net/wireless", "grep wlan0"])
 def get_rpi_power_status():
     return _underVoltage.get()
 
+def get_host_name():
+    return socket.gethostname()
+
+def get_host_ip():
+    return socket.gethostbyname(socket.gethostname())
```
Comment (Sennevds) [inline src/system_sensors.py:186]: This returns 127.0.1.1 on Raspbian when the hostname is changed by using the raspi-config tool(this puts the hostname in the /etc/hosts file)

Comment (MaxVRAM) [reply src/system_sensors.py:186]: Good spotting. I'll look into it.

**Comment by MaxVRAM** [comment]
@Sennevds  sorry about that - still working my way around Git commands. New commit should be now added to this PR

**Review by Sennevds — APPROVED** [review]

**Comment by Sennevds** [comment]
> @Sennevds sorry about that - still working my way around Git commands. New commit should be now added to this PR

No problem I'm just happy people are using and improving my code!

**Pull request merged**